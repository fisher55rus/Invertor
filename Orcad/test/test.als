.ALIASES
V_V1            V1(+=$N_0001 -=0 )
R_R1            R1(1=$N_0001 2=$N_0002 )
R_R2            R2(1=$N_0002 2=$N_0003 )
R_R3            R3(1=$N_0005 2=$N_0004 )
E_LIMIT1          LIMIT1(OUT=$N_0005 IN=$N_0003 )
X_U1A           U1A(+=0 -=$N_0002 V+=+15 V-=-15 OUT=$N_0003 )
R_R4            R4(1=I_sence 2=$N_0006 )
X_U2A           U2A(+=$N_0004 -=$N_0006 V+=+15 V-=-15 OUT=$N_0007 )
C_C1            C1(1=$N_0006 2=0 )
R_R5            R5(1=$N_0004 2=$N_0007 )
R_R6            R6(1=$N_0007 2=$N_0008 )
C_C2            C2(1=$N_0008 2=0 )
R_R7            R7(1=$N_0009 2=+15 )
R_R8            R8(1=0 2=$N_0009 )
R_R9            R9(1=$N_0010 2=0 )
R_R10           R10(1=-15 2=$N_0010 )
R_R11           R11(1=0 2=bb )
X_U3A           U3A(+=$N_0008 -=$N_0009 V+=+15 V-=-15 OUT=bb )
R_R12           R12(1=0 2=aa )
X_U4A           U4A(+=$N_0010 -=$N_0008 V+=+15 V-=-15 OUT=aa )
R_R13           R13(1=$N_0012 2=$N_0011 )
R_R14           R14(1=$N_0014 2=$N_0013 )
E_E1            E1(3=$N_0011 4=$N_0015 1=aa 2=0 )
L_L1            L1(1=0 2=I_sence )
R_R15           R15(1=I_sence 2=0 )
M_M1            M1(d=$N_0016 g=$N_0012 s=$N_0015 s=$N_0015 )
D_D1            D1(1=$N_0015 2=$N_0016 )
E_E2            E2(3=$N_0013 4=0 1=bb 2=0 )
V_V2            V2(+=$N_0016 -=0 )
M_M2            M2(d=$N_0015 g=$N_0014 s=0 s=0 )
D_D2            D2(1=0 2=$N_0015 )
C_C3            C3(1=$N_0017 2=0 )
L_L2            L2(1=$N_0017 2=0 )
M_M3            M3(d=$N_0016 g=$N_0019 s=$N_0018 s=$N_0018 )
M_M4            M4(d=$N_0018 g=$N_0020 s=0 s=0 )
E_E3            E3(3=$N_0021 4=$N_0018 1=bb 2=0 )
E_E4            E4(3=$N_0022 4=0 1=aa 2=0 )
R_R16           R16(1=$N_0020 2=$N_0022 )
R_R17           R17(1=$N_0019 2=$N_0021 )
D_D3            D3(1=0 2=$N_0018 )
D_D4            D4(1=$N_0018 2=$N_0016 )
L_L3            L3(1=$N_0023 2=$N_0018 )
L_L4            L4(1=$N_0024 2=$N_0023 )
L_L5            L5(1=$N_0015 2=$N_0024 )
R_R18           R18(1=$N_0017 2=0 )
V_V3            V3(+=-15 -=0 )
V_V4            V4(+=+15 -=0 )
Kn_K1            K1()
Kn_K2            K2()
_    _(I_sence=I_sence)
_    _(bb=bb)
_    _(aa=aa)
_    _(-15=-15)
_    _(+15=+15)
.ENDALIASES
