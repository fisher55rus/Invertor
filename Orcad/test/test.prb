[DISPLAYS]
BEGIN DISPLAY LAST SESSION
ANALYSIS TRANSIENT_ANALYSIS
SYMBOL ALWAYS
TRACECOLORSCHEME NORMAL
BEGIN ANAPLOT 1
XBASE
BEGIN XAXIS
RANGEFLAG AUTO
TYPE LINEAR
UNIT s
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE DOTS
MAJORINTERSECTMAJOR
MAJORINTERSECTMINOR
MINORSTYLE DOTS
MINORINTERSECTMAJOR
MINORINTERSECTMINOR
END GRIDS
END XAXIS
BEGIN YAXIS 1
ACTIVE
RANGEFLAG AUTO
TYPE LINEAR
UNIT V
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE DOTS
MAJORINTERSECTMAJOR
MAJORINTERSECTMINOR
MINORSTYLE DOTS
MINORINTERSECTMAJOR
MINORINTERSECTMINOR
END GRIDS
BEGIN TRACE V(I_sence)
MARKERID 0
END TRACE V(I_sence)
END YAXIS 1
END ANAPLOT 1
BEGIN ANAPLOT 2
BEGIN XAXIS
RANGEFLAG AUTO
TYPE LINEAR
UNIT s
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE DOTS
MAJORINTERSECTMAJOR
MAJORINTERSECTMINOR
MINORSTYLE DOTS
MINORINTERSECTMAJOR
MINORINTERSECTMINOR
END GRIDS
END XAXIS
BEGIN YAXIS 1
ACTIVE
RANGEFLAG AUTO
TYPE LINEAR
UNIT V
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE DOTS
MAJORINTERSECTMAJOR
MAJORINTERSECTMINOR
MINORSTYLE DOTS
MINORINTERSECTMAJOR
MINORINTERSECTMINOR
END GRIDS
BEGIN TRACE V(LIMIT1:IN)
MARKERID 0
END TRACE V(LIMIT1:IN)
END YAXIS 1
END ANAPLOT 2
BEGIN ANAPLOT 3
BEGIN XAXIS
RANGEFLAG AUTO
TYPE LINEAR
UNIT s
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE DOTS
MAJORINTERSECTMAJOR
MAJORINTERSECTMINOR
MINORSTYLE DOTS
MINORINTERSECTMAJOR
MINORINTERSECTMINOR
END GRIDS
END XAXIS
BEGIN YAXIS 1
ACTIVE
RANGEFLAG AUTO
TYPE LINEAR
UNIT A
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE DOTS
MAJORINTERSECTMAJOR
MAJORINTERSECTMINOR
MINORSTYLE DOTS
MINORINTERSECTMAJOR
MINORINTERSECTMINOR
END GRIDS
BEGIN TRACE I(L5)
MARKERID 0
END TRACE I(L5)
END YAXIS 1
END ANAPLOT 3
BEGIN ANAPLOT 4
BEGIN XAXIS
RANGEFLAG AUTO
TYPE LINEAR
UNIT s
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE DOTS
MAJORINTERSECTMAJOR
MAJORINTERSECTMINOR
MINORSTYLE DOTS
MINORINTERSECTMAJOR
MINORINTERSECTMINOR
END GRIDS
END XAXIS
BEGIN YAXIS 1
ACTIVE
RANGEFLAG AUTO
TYPE LINEAR
UNIT V
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE DOTS
MAJORINTERSECTMAJOR
MAJORINTERSECTMINOR
MINORSTYLE DOTS
MINORINTERSECTMAJOR
MINORINTERSECTMINOR
END GRIDS
BEGIN TRACE V(V1:+)
MARKERID 0
END TRACE V(V1:+)
END YAXIS 1
END ANAPLOT 4
BEGIN ANAPLOT 5
ACTIVE
BEGIN XAXIS
RANGEFLAG AUTO
TYPE LINEAR
UNIT s
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE DOTS
MAJORINTERSECTMAJOR
MAJORINTERSECTMINOR
MINORSTYLE DOTS
MINORINTERSECTMAJOR
MINORINTERSECTMINOR
END GRIDS
END XAXIS
BEGIN YAXIS 1
ACTIVE
RANGEFLAG NORANGE
TYPE LINEAR
UNIT  
END YAXIS 1
END ANAPLOT 5
END DISPLAY LAST SESSION
BEGIN DISPLAY LAST DISPLAY
ANALYSIS TRANSIENT_ANALYSIS
SYMBOL ALWAYS
TRACECOLORSCHEME NORMAL
BEGIN ANAPLOT 1
ACTIVE
XBASE
BEGIN XAXIS
RANGEFLAG AUTO
TYPE LINEAR
UNIT s
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE DOTS
MAJORINTERSECTMAJOR
MAJORINTERSECTMINOR
MINORSTYLE DOTS
MINORINTERSECTMAJOR
MINORINTERSECTMINOR
END GRIDS
END XAXIS
BEGIN YAXIS 1
ACTIVE
RANGEFLAG NORANGE
TYPE LINEAR
UNIT  
END YAXIS 1
END ANAPLOT 1
END DISPLAY LAST DISPLAY
