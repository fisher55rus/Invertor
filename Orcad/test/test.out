
**** 04/13/19 10:37:14 ********* PSpice 9.2 (Mar 2000) ******** ID# 1 ********

 * D:\GIT\MAI\Invertor\Orcad\5_full1\test.sch


 ****     CIRCUIT DESCRIPTION


******************************************************************************




* Schematics Version 9.2
* Sat Apr 13 10:27:33 2019



** Analysis setup **
.tran 0ns 10m 0 1.57u SKIPBP
.LIB "D:\GIT\MAI\Invertor\Orcad\1_Power_cascade\1STK.lib"


* From [PSPICE NETLIST] section of d:\Program Files\Orcad\PSpice\PSpice.ini:
.lib "D:\Program Files\Orcad\PSpice\UserLib\OPA_2.lib"
.lib "D:\Program Files\Orcad\PSpice\UserLib\MyLib.lib"
.lib "nom.lib"

.INC "test.net"



**** INCLUDING test.net ****
* Schematics Netlist *



V_V1         $N_0001 0  
+SIN 0 5 400 0 0 0
R_R1         $N_0001 $N_0002  5k  
R_R2         $N_0002 $N_0003  100k  
R_R3         $N_0005 $N_0004  1k  
E_LIMIT1         $N_0005 0 VALUE {LIMIT(V($N_0003),-1,1)}
X_U1A         0 $N_0002 +15 -15 $N_0003 TL082
R_R4         I_sence $N_0006  1k  
X_U2A         $N_0004 $N_0006 +15 -15 $N_0007 TL082
C_C1         $N_0006 0  100p  
R_R5         $N_0004 $N_0007  100k  
R_R6         $N_0007 $N_0008  10k  
C_C2         $N_0008 0  100p  
R_R7         $N_0009 +15  10k  
R_R8         0 $N_0009  10k  
R_R9         $N_0010 0  10k  
R_R10         -15 $N_0010  10k  
R_R11         0 bb  10k  
X_U3A         $N_0008 $N_0009 +15 -15 bb TL082
R_R12         0 aa  10k  
X_U4A         $N_0010 $N_0008 +15 -15 aa TL082
R_R13         $N_0012 $N_0011  10  
R_R14         $N_0014 $N_0013  10  
E_E1         $N_0011 $N_0015 aa 0 1
L_L1         0 I_sence  5H  
R_R15         I_sence 0  33  
M_M1         $N_0016 $N_0012 $N_0015 $N_0015 IRF255
D_D1         $N_0015 $N_0016 Dbreak-X 
E_E2         $N_0013 0 bb 0 1
V_V2         $N_0016 0 18
M_M2         $N_0015 $N_0014 0 0 IRF255
D_D2         0 $N_0015 Dbreak-X 
C_C3         $N_0017 0  5u  
L_L2         $N_0017 0  12mH  
M_M3         $N_0016 $N_0019 $N_0018 $N_0018 IRF255
M_M4         $N_0018 $N_0020 0 0 IRF255
E_E3         $N_0021 $N_0018 bb 0 1
E_E4         $N_0022 0 aa 0 1
R_R16         $N_0020 $N_0022  10  
R_R17         $N_0019 $N_0021  10  
D_D3         0 $N_0018 Dbreak-X 
D_D4         $N_0018 $N_0016 Dbreak-X 
L_L3         $N_0023 $N_0018  50uH  
L_L4         $N_0024 $N_0023  1mH  
L_L5         $N_0015 $N_0024  5uH  
R_R18         $N_0017 0  0.1  
V_V3         -15 0 -15
V_V4         +15 0 15
Kn_K1         L_L5 L_L1     1
Kn_K2         L_L4 L_L2     1

**** RESUMING test.cir ****
.PROBE V(*) I(*) W(*) D(*) NOISE(*) 


.END

**** 04/13/19 10:37:14 ********* PSpice 9.2 (Mar 2000) ******** ID# 1 ********

 * D:\GIT\MAI\Invertor\Orcad\5_full1\test.sch


 ****     Diode MODEL PARAMETERS


******************************************************************************




               Dbreak-X        X_U1A.dx        X_U1A.dy        X_U2A.dx        
          IS  100.000000E-12  800.000000E-18  800.000000E-18  800.000000E-18 
          RS     .05            1               1.000000E-03    1            
         CJO  150.000000E-12                   10.000000E-12                 


               X_U2A.dy        X_U3A.dx        X_U3A.dy        X_U4A.dx        
          IS  800.000000E-18  800.000000E-18  800.000000E-18  800.000000E-18 
          RS    1.000000E-03    1               1.000000E-03    1            
         CJO   10.000000E-12                   10.000000E-12                 


               X_U4A.dy        
          IS  800.000000E-18 
          RS    1.000000E-03 
         CJO   10.000000E-12 


**** 04/13/19 10:37:14 ********* PSpice 9.2 (Mar 2000) ******** ID# 1 ********

 * D:\GIT\MAI\Invertor\Orcad\5_full1\test.sch


 ****     Junction FET MODEL PARAMETERS


******************************************************************************




               X_U1A.jx        X_U2A.jx        X_U3A.jx        X_U4A.jx        
               NJF             NJF             NJF             NJF             
         VTO   -1              -1              -1              -1            
        BETA  984.200000E-06  984.200000E-06  984.200000E-06  984.200000E-06 
          IS    2.500000E-12    2.500000E-12    2.500000E-12    2.500000E-12 


**** 04/13/19 10:37:14 ********* PSpice 9.2 (Mar 2000) ******** ID# 1 ********

 * D:\GIT\MAI\Invertor\Orcad\5_full1\test.sch


 ****     MOSFET MODEL PARAMETERS


******************************************************************************




               IRF255          
               NMOS            
       LEVEL    3            
           L    2.000000E-06 
           W   28            
         VTO    3.686        
          KP    1.040000E-06 
       GAMMA    0            
         PHI     .6          
      LAMBDA    0            
          RD    1.000000E-03 
          RS     .01         
          RG    7.008        
          RB    6.906000E-03 
         RDS    1.000000E+06 
          IS  562.200000E-09 
          JS    0            
           N    1.833        
          PB    2.532        
        PBSW    2.532        
         CBD    5.886000E-09 
          CJ    0            
        CJSW    0            
          MJ    1.089        
        CGSO   43.180000E-12 
        CGDO    7.890000E-12 
        CGBO    0            
         TOX    2.000000E-06 
          XJ    0            
       UCRIT   10.000000E+03 
       DELTA    0            
         ETA    0            
      DIOMOD    1            
         VFB    0            
        LETA    0            
        WETA    0            
          U0    0            
        TEMP    0            
         VDD    0            
       XPART    0            



          JOB CONCLUDED

          TOTAL JOB TIME            8.10
